src/main/java/com/rakovets/course/java/core/practice/arrays/Task03.java

```diff
@@ -57,8 +84,15 @@ static int[] getMinMarks(int[][] marks) {
      * @return максимальная отметка
      */
     static int[] getMaxMarks(int[][] marks) {
-        //TODO
-        // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return null;
+        int maxMark[] = new int[marks.length];
+        for (int i = 0; i < marks.length; i++) {
+            int maxMarkInRow = marks[i][0];
+            for (int j = 0; j < marks[0].length; j++) {
+                if (maxMarkInRow < marks[i][j]) {
+                    maxMarkInRow = marks[i][j];
+                }
+            } maxMark[i] = maxMarkInRow;
```

Comment:
"Violation: There aren't correct indentations for inner code
Solution: Add correct indentations for inner code"